kit.yml

```diff
@@ -212,4 +212,3 @@ provided:
         project:
           sensitive: false
           prompt:    OpenStack project
-  openstack: *provided_openstack
```

Comment:
This should have been openstack-cpi, not a second openstack: (again, my original fault)